### DISCUSSION TITLE
Problem Configure your Python environment with a Jupyter Notebook on Google Colab (Chicago Crime Analysis - Level 1)

### DISCUSSION THREAD

**Discussion by imanshahmari** [DevSkillsHQ/discusssions#15] [Help] [answered]
Get this when i click on the link to visit the url

![Screenshot 2023-03-28 151323](https://user-images.githubusercontent.com/75856104/228287846-64d90043-faa7-416c-b084-607dc9cfb7c4.png)
![Screenshot 2023-03-28 153416](https://user-images.githubusercontent.com/75856104/228287910-7af7d4ce-98c7-4667-b4ae-ea83673a1e0e.png)
![Screenshot 2023-03-28 153618](https://user-images.githubusercontent.com/75856104/228288192-69f0600b-07a7-48e6-8260-0e808d8a7774.png)

**Comment by fenske** [comment]
Hi @imanshahmari . I'm Anton from DevSkills. Happy to assist!

Could you please clarify what precise link you're referring to? (If possible, please share a screencast that shows how it's failing).

**Reply by imanshahmari** [reply]
Hello Anton
Im trying to query the database with the following code:

import pandas as pd
sample_count = 200
df = pd.io.gbq.read_gbq('''
  select year, count(*) as crimes, avg(case when arrest then 1.000 else 0 end) as arrest_rate FROM `bigquery-public-data.chicago_crime.crime` group by 1 order by 3 desc'''
, project_id='chicago-382013', dialect='standard')
df.head()

I get this: 
Please visit this URL to authorize this application: https://accounts.google.com/o/oauth2/auth?response_type=code&client_id=725825577420-unm2gnkiprugilg743tkbig250f4sfsj.apps.googleusercontent.com&redirect_uri=https%3A%2F%2Fpydata-google-auth.readthedocs.io%2Fen%2Flatest%2Foauth.html&scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fbigquery&state=PVoruwU8tYF2cbhevBkNbxxGVxerOv&access_type=offline

When I click on the link I get:

![Screenshot 2023-03-29 123425](https://user-images.githubusercontent.com/75856104/228507743-862fb4fc-e6a4-464e-b41e-5fbb81665c0e.png)

**Comment by imanshahmari** [comment]
I have not given the google cloud platform any payment information because they charge your credit card after the free trial and I don't think its necessary to link my bank card for doing a test. Maybe this is why I don't have access?

**Reply by fenske** [reply]
Ok, understood. Let's do it this way not to waste your time. I'll set it up for you but I'll need to know your google email address. Could you share it?

**Reply by imanshahmari** [reply]
yes here it is:
imanshahmari@gmail.com

**Reply by fenske** [reply, marked as answer]
Sorry for keeping you waiting @imanshahmari. 

While creating a Notebook ourselves, we've encountered the same issue you ran into, and it's apparently related to some general changes that have been done to how Google Colab communicates with BigQuery. And unfortunately, it looks like we'll need to come up with a different way for candidates to submit their solutions.

In the meantime, I've updated the instructions for how to submit your solutions. Please find them in the README file of your assignment repository.

Please let me know if you have any other questions. And once again, my apologies for this delay.

I wish you the best of luck with the rest of your interview process.
